mls-rs-provider-sqlite/src/lib.rs

```diff
@@ -170,4 +216,24 @@ mod tests {
 
         assert_eq!(current_schema, 1);
     }
+
+    #[test]
+    pub fn journal_mode_test() {
+        let db_path = temp_dir().join("test_db.sqlite");
```

Comment:
As linked previously, this should use `tempdir` from the `tempfile` crate, which we already depend on. `std::env::temp_dir` has several issues:
- No clean-up is performed, or at least it is platform-dependent.
- Other programs may use it and files may conflict.

`tempdir` avoids all this trouble by creating a randomly named directory and taking care of deleting it.